modelchecker/BUILD.bazel

```diff
@@ -22,15 +22,13 @@ go_library(
     visibility = ["//visibility:public"],
     deps = [
         "//lib",
-        "//proto",
-        "//proto:options",
+        "//proto:proto_go_proto",
         "@com_github_golang_glog//:glog",
-        "@com_github_jayaprabhakar_go_clone//:go-clone",
-        "@org_golang_google_protobuf//encoding/protojson:go_default_library",
-        "@org_golang_google_protobuf//proto:go_default_library",
+        "@com_github_huandu_go_clone//:go-clone",
```

Comment:
@tekumara One reason I had to fork the cloning library was it was missing a feature to override pointer references. 
https://github.com/huandu/go-clone/issues/23

Reply:
It's still using your fork - see the replace statement in go.mod 

I had to do it like this to make go.mod work

Reply:
https://go.dev/ref/mod#go-mod-file-replace

Reply:
I'm assuming bazel honours this but admittedly I haven't tested this - is there a way to test this?